onfleet/exceptions.py

```diff
@@ -1,3 +1,13 @@
+import re
+
+options_re = re.compile(r'"(.*?)"')
+
+
+def parse_options(self):
+    """Parse the onfleet string for the proposed options."""
+    return options_re.findall(self.cause)
```

Comment:
Given that this doesn't really truly have anything to do with exceptions (it's for parsing an error response from an API, but it's not for parsing an exception) I think it actually belongs in the onfleet.py file.